packages/expo/bin/cli.ts

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+import arg from 'arg';
+import chalk from 'chalk';
+
+const defaultCmd = 'config';
+
+export type Command = (argv?: string[]) => void;
+
+const commands: { [command: string]: () => Promise<Command> } = {
+  // Add a new command here
+  config: () => import('../cli/config').then((i) => i.expoConfig),
+};
+
+const args = arg(
+  {
+    // Types
+    '--version': Boolean,
+    '--help': Boolean,
+
+    // Aliases
+    '-v': '--version',
+    '-h': '--help',
+  },
+  {
+    permissive: true,
+  }
+);
+
+if (args['--version']) {
+  // Version is inlined into the file using taskr build pipeline
+  console.log(`expo v${process.env.__EXPO_VERSION}`);
+  process.exit(0);
+}
+
+// Check if we are running `npx expo <subcommand>` or `npx expo`
+const targetCmd = Boolean(commands[args._[0]]);
+
+// Handle `--help` flag
+if (!targetCmd && args['--help']) {
```

Comment:
Should the help also be triggered when you just run `npx expo`, like `npx eas-cli`? (as in, forget to add a command). I think that might be the preferred behavior, not just when running `npx expo --help`.

Reply:
Ah nvm, you made `npx expo config` the default command 😄